scripts/build_palettes.ts

```diff
@@ -22,41 +22,41 @@ await Promise.all(
 
 Promise.all(
   Object.entries(flavors).flatMap(async ([identifier, { name, colors }]) => {
-    const fname = identifier.charAt(0).toUpperCase() + identifier.slice(1);
+    const safeName = identifier.charAt(0).toUpperCase() + identifier.slice(1);
```

Comment:
I feel like `safeName` makes the code harder to read?
How about `flavourName`? Since that's what it actually is.

Reply:
But it isn't just the flavor name? The whole point is that this variable is the capitalized proper name, minus any accents/extras.

Reply:
Sure but it doesn't matter in this scenario really? Like we're not using an `unsafeName` anywhere else. Feels like it's just nicer saying `flavourName` or if you really want `safeFlavourName`

Reply:
But then we would have `flavorName` and `name` in the same context? How are you supposed to differentiate there? We *can* make `name` `unsafeName` if you wish!

Reply:
The only safe/unsafe part is related to the accents right, but `name` doesn't even have the accents right? Feels like it'd make more sense to do `lowerName` and `upperName`?

Reply:
`name` does have accents 😅

Reply:
You are thinking of `identifier`, which `safeName` uses here as a version without the accent which it then capitalizes to look like `name` minus the accent.

Reply:
Personally, I've grown to slightly dislike `safeName` without any sort of comment or obvious context on why it's "safe" from my experience at work. In any case, I didn't really intend for this to be a blocker on the PR so I'm happy to leave as is.

Reply:
I mean I can do `nameWithoutAccent` if you really want to be specific.